Package.swift

```diff
@@ -0,0 +1,47 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "AccessibilitySnapshot",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v11),
```

Comment:
Let's make it 12 to match the current CocoaPods config. I think it will be easier to have the same minimum version across the board.
```suggestion
        .iOS(.v12),
```